blocks/targeted-block/targeted-block.js

```diff
@@ -1,86 +1,26 @@
 import { events } from '@dropins/tools/event-bus.js';
-import * as Cart from '@dropins/storefront-cart/api.js';
-import { fetchGraphQl } from '@dropins/tools/fetch-graphql.js';
+import { getActiveRules, getCatalogPriceRules } from './qraphql.js';
+import conditionsMatched from './condition-matcher.js';
 import { readBlockConfig } from '../../scripts/aem.js';
 import { loadFragment } from '../fragment/fragment.js';
 
 const blocks = [];
 const displayedBlockTypes = [];
 
-const getActiveRules = async () => {
-  try {
-    const response = await fetchGraphQl(
-      `query CUSTOMER_SEGMENTS($cartId: String!){
-          customerSegments(cartId: $cartId) {
-            name
-          }
-          CustomerGroup {
-            name
-          }
-          cart(cart_id: $cartId) {
-            rules {
-              name
-            }
-          }
-        }
-      `,
-      {
-        method: 'GET',
-        variables: {
-          cartId: Cart.getCartDataFromCache().id,
-        },
-      },
-    );
-    return response.data;
-  } catch (error) {
-    console.error('Could not retrieve customer segments', error);
-  }
-  return [];
+const getSkuFromUrl = () => {
```

Comment:
Is it possible to get sku from pdp event?

Reply:
Still not resolved

Reply:
Resolved.